### PR TITLE
Delete conatct/organization  from store

### PR DESCRIPTION
**Pull request by JamesSCBD**

### CODE REVIEW

File: store/contacts.js [hunk 1]
```diff
@@ -89,6 +89,11 @@ const $actions = {
   clearSelection({ commit }) {
     commit(CLEAR_SELECTION);
   },
+
+  async deleteContact({ commit }, contact) {
```
Comment (sbilodeau) [inline store/contacts.js:93]: The parameter must be `contactId` Not the entire object

File: store/organizations.js [hunk 2]
```diff
@@ -137,6 +137,11 @@ const $actions = {
   clearSelection({ commit }) {
     commit(CLEAR_SELECTION);
   },
+
+  async deleteOrganization({ commit }, organization) {
```
Comment (sbilodeau) [inline store/organizations.js:141]: Same. Parameter must be `organizationId` and not the entire object

File: store/contacts.js (hunk 1 above)
Comment (JamesSCBD) [reply store/contacts.js]: Done!

File: store/organizations.js (hunk 2 above)
Comment (JamesSCBD) [reply store/organizations.js]: Done!

**Pull request merged**